packages/web3-auth/src/SocialLogin.tsx

```diff
@@ -54,25 +54,31 @@ class SocialLogin {
     return whiteListUrlResponse.data
   }
 
-  async init(
-    chainId: string = defaultSocialLoginConfig.defaultChainId,
-    whitelistUrls?: { [P in string]: string },
-    network: 'mainnet' | 'testnet' = 'testnet'
-  ) {
+  async init(socialLoginDTO?: Partial<SocialLoginDTO>) {
+    const finalDTO: SocialLoginDTO = {
+      chainId: '0x1',
+      whitelistUrls: {},
+      network: 'testnet'
+    }
+    if (socialLoginDTO) {
+      if (socialLoginDTO.chainId) finalDTO.chainId = socialLoginDTO.chainId
```

Comment:
Under what circumstances this chainId needs to be overridden?? and how does 'testnet' value of network field changes the EOA account generated by social login sdk?? 
@AmanRaj1608 @livingrockrises

Reply:
- On web3Auth Initialisation they expect a chainId cause they generate a provider and signer related to that chainId only. 
- after sign in with soicals, the web3auth servers to which it redirects changes on what we pass, which changes the EOA.

Reply:
Would this change in chainId change the generated EOA address as well?

Reply:
no chainId is used to give back the provider only

Reply:
network values can be


<img width="546" alt="image" src="https://user-images.githubusercontent.com/42104907/209351927-7a1bb8f1-96d3-49c3-8362-74dd79d19591.png">